### DISCUSSION TITLE
Τελική αναφορά - Παραδοτέο 10 Π2018134

### DISCUSSION THREAD

**Discussion by xristinasp** [courses-ionio/sw#1756] [Show and tell]
- [Η αναφορά μου](https://github.com/xristinasp/sw/tree/2018134/projects/2018134)

**Comment by epidrome** [comment]
@xristinasp σύμφωνα με τις οδηγίες θα πρέπει να γίνει αίτημα ενσωμάτωσης και να έχει γίνει προφορική εξέταση για να γίνει δεκτό

**Reply by xristinasp** [reply]
@epidrome Επειδη έχω ξενοικιάσει απο την Κέρκυρα δεν ήταν δυνατό να έρθω νωρίτερα να παρουσιάσω σε εσας, οπότε θα γίνει η παρουσίαση στον κ. Βραχάτη την ημέρα της εξέτασης του μαθήματος και θα κάνω τότε το αίτημα ενσωμάτωσης (μαζί με τα σχόλια του κ. Βραχάτη) γιατί αν έχω καταλάβει σωστά χωρίς τελική παρουσίαση το pull request θα απορριφθεί. Ή πρέπει να κάνω από τώρα το pull request και να προσθέσω μετά τα σχόλια του κ. Βραχάτη;

**Reply by epidrome** [reply]
σύμφωνα με τις οδηγίες τα εκπρόθεσμα αιτήματα βαθμολογούνται τον Σεπτέμβριο
https://epidrome.github.io/teaching/progress/
